docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
 
-Finally, if you ran Alpha Rarefaction, you will also have a link to interactive
-plots that can be used to show how different measures of alpha diversity
-correlate with different metadata categories:
+.. figure::  images/observed_OTU_workflow.png
+   :align:   center
+
+Click the run button to start the process of the alpha diversity analysis. The view will return to the original screen, while the alpha diversity analysis job runs. 
+
+Another type of alpha diversity analysis is the Shannon diversity index. This analyzes the amount of taxa per the total amount of taxa. It takes into account both diversity as well as abundance.  
 
-.. figure::  images/analysis-closed-ref-alpha.png
+To perform an Shannon diversity index, select the rarefied "dflt_name - BIOM" artifact in the processing network and select "Process". Then select the "dflt_name - BIOM" artifact and select "alpha_diversity" from the drop-down menu. The parameters will appear below the workflow diagram: 
+
+.. figure::  images/shannon_diversity_parameter.png
    :align:   center
 
+Several parameters have been automatically selected for you. In the "p-metric" field enter "shannon", and click "Add Command". Once the command is added the workflow should appear as follows:
 
-Analysis of deblur processing
------------------------------
+.. figure::  images/shannon_diversity_workflow.png
+   :align:   center
 
-Creating an analysis of your deblurred data is virtually the same as the
-process for the Closed Reference data, but there are a few quirks.
+Click the run button to start the process of the alpha diversity analysis. The view will return to the original screen, while the alpha diversity analysis job runs. 
+
+The final alpha diversity analysis is Faith’s phylogenetic diversity index. This index also measured abundance and diversity but displays it in tree form rather than in a plot.
+
+To perform a Faith's phylogenetic diversity index, select the rarefied "dflt_name - BIOM" artifact in the processing network and select "Process". Then select the "dflt_name - BIOM" artifact and  select "alpha_diversity" from the drop-down menu. The parameters will appear below the workflow diagram: 
+
+.. figure::  images/faith_pd_parameter.png
+   :align:   center
 
-First, because the deblur process creates two separate BIOM tables, you'll want
-to make a note of the specific object ID number for the artifact you want to
-use. In my case, that's ID 26017, the deblurred table with 'only-16s' reads.
+Several parameters have been automatically selected for you. In the "p-metric" field enter "faith_pd" and in the "i-tree" field select "/databases/gg/13_8/trees/97_otus_no_none.tree". Then click "Add Command" and once the command is added the workflow should appear as follows:
```

Comment:
IMO, yes, let the users decide it's too tedious and skip ahead rather than assume they get the idea and leave them with too few instructions. I've done a lot of training, and the repetition is key for some people.